tox.ini

```diff
@@ -33,18 +33,18 @@ commands = {posargs}
 setenv = VIRTUALENV_DIR = {envdir}
```

Comment:
Ah, I can leave it here then, we just need to hook it up somewhere (Travis?), otherwise it will get even more out of date than it currently is.